docs/tutorial/pixabay-source-plugin-tutorial.md

```diff
@@ -1,9 +1,7 @@
 ---
-title: Pixabay Image Source Plugin Tutorial
+title: "Creating an Image Source Plugin"
```

Comment:
If we're changing this title should the link titles elsewhere in the site also be updated?

Reply:
We certainly could, I kept different context in other places but let this title be shorter since the body of the text mentions Pixabay.

Reply:
Sounds good.

Reply:
I just pushed an update that makes the tutorial title more descriptive, but I also found there was precedent for referring to this tutorial as a "source plugin tutorial" without further qualifying. So I'm going to leave the rest of the links as-is.

Reply:
Scratch that, I was 10 minutes too slow to push an update! I think the PR was fine enough to leave it alone though.

Reply:
Drat! Sorry.